team-members/sasha-gorodnya.html

```diff
@@ -0,0 +1,98 @@
+       <nav>
+         <ul>
+           <li>
+               <a href="#part1">About me</a>
+           </li>
+           <li>
+               <a href="#part2">Positive qualities about myself</a>
+           </li>
+           <li>
+               <a href="#part3">To-do list with my friends</a>
+           </li>
+            <li>
+               <a href="#part4">Contact me</a>
+           </li>
+         </ul>
+       </nav>
+        <h1><section id="part1"><i>About me</i></section></h1>
+      <img src="../images/sasha-gor.jpg"
+       width="400" height="400" alt="my_photo">
+    </header>
+    <main>
+      <section>
+        <p>Let me introduce myself. My name is Sasha. I was born in the 24 th of August, 1998 in Kiev.<br>
+        I am twenty-one years old. I am a fourth-year student. I study at Kyiv National Economic University <br>named after Vadym Hetman.
+          My first job was as manager of the tourism.<br>I’m now ready for a more challenging assignment and decide to learn programming.<br>
+         </p>
+        <br>
+        <h2><section id="part2"><i>Positive qualities about myself:)</i></section></h2>
+        <ol>
+          <li>I will complete my task as soon as possible without delay or excuses.</li>
+          <li>I highly interested in self- growth and obtaining knowledge.</li>
+          <li>I keep everything as a positive even though its make me fail.</li>
+          <li>I will not hesitate to help people when I know how to help. <br>
+           If I able to help people, I will surely help without asking any rewards.</li>
+          <li>I won’t hesitate to start communicate.</li>
+          <li>I always keep my imagination too high.</li>
+        </ol>
+      </section>
+      <section>
+            <h2><section id="part3"><i>To-do list with my friends</i></section></h2>
+            <img src="" alt="">
+        <table>
+          <thead>
+            <tr>
+            <th>Mon.</th>
+            <th>Tues.</th>
+            <th>Wed.</th>
+            <th>Thurs</th>
+            <th>Fri.</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Take a photo</td>
+              <td>Go to the cafeteria</td>
+              <td>Take acting classes</td>
+              <td>Go out of town</td>
+              <td>Gym</td>
+            </tr>
+          </tbody>
+          <tfoot>
+            <tr>
+            <td>1</td>
+            <td>2</td>
+            <td>3</td>
+            <td>4</td>
+            <td>5</td>
+            </tr>
+          </tfoot>
+        </table>
+       </section>
+  </main>
+     <footer>
+    <h4>Сonnect with me on Instagram:</h4>
+    <button class="btn btn-success"
+   onclick=" window.open('https://www.instagram.com/sandelfini/?hl=uk','_blank')">Instagram</button>
+   <form action="/action_page.php"><br>
+     <fieldset>
+       <legend><strong><section id="part4">Contact me:</section></strong></legend>
+       <form action="/action_page.php" autocomplete="on">
```

Comment:
form shouldn't be inside the other form + this tag has no closing tag

Reply:
done